### PR TITLE
extend requirements for optional dependencies

### LINKED ISSUE #5: Missing requirements
Hi,

```
 % pew mktmpenv
Using base prefix '/usr'
New python executable in /home/fr/.local/share/virtualenvs/e5ecf64498bb9ee/bin/python3
Also creating executable in /home/fr/.local/share/virtualenvs/e5ecf64498bb9ee/bin/python
Installing setuptools, pip, wheel...done.
This is a temporary environment. It will be deleted when you exit
Launching subshell in virtual environment. Type 'exit' or 'Ctrl+D' to return.
 % pip install partd
Collecting partd
Collecting locket (from partd)
Installing collected packages: locket, partd
Successfully installed locket-0.2.0 partd-0.3.2
 % python -c 'import partd'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/fr/.local/share/virtualenvs/e5ecf64498bb9ee/lib/python3.5/site-packages/partd/__init__.py", line 3, in <module>
    from .file import File
  File "/home/fr/.local/share/virtualenvs/e5ecf64498bb9ee/lib/python3.5/site-packages/partd/file.py", line 3, in <module>
    from .core import Interface
  File "/home/fr/.local/share/virtualenvs/e5ecf64498bb9ee/lib/python3.5/site-packages/partd/core.py", line 7, in <module>
    from toolz import memoize
ImportError: No module named 'toolz'
```

and after I install toolz, pandas is missing.

With toolz and pandas, the import goes well.

### PR DESCRIPTION
**Pull request by mrocklin**
Fixes #5 

cc @sciunto

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,4 +15,5 @@
       install_requires=list(open('requirements.txt').read().strip().split('\n')),
       long_description=(open('README.rst').read() if exists('README.rst')
                         else ''),
+      extras_require = {'complete': ['numpy', 'pandas', 'zmq', 'blosc']}
```
Comment (sciunto) [inline setup.py]: missing coma :)

Comment (mrocklin) [inline setup.py]: Whoops!  Fixed.

On Thu, Feb 11, 2016 at 6:00 PM, François Boulogne <notifications@github.com

**Comment by sciunto** [comment]
Humm, travis is not happy with py26. Not obvious to me.

**Comment by mrocklin** [comment]
Blosc, a somewhat fringe dependency apparently has some issues.  I honestly
don't mind skipping blosc tests in 2.6.

On Thu, Feb 11, 2016 at 6:14 PM, François Boulogne <notifications@github.com

**Comment by sciunto** [comment]
me neither

**Comment by mrocklin** [comment]
Turned off blosc installation in travis.ci

**Comment by sciunto** [comment]
I think we are good. :)

**Pull request merged**